src/cryptography/hazmat/backends/openssl/dsa.py

```diff
@@ -7,32 +7,14 @@
 from cryptography import utils
 from cryptography.exceptions import InvalidSignature
 from cryptography.hazmat.backends.openssl.utils import (
-    _calculate_digest_and_algorithm, _truncate_digest
+    _calculate_digest_and_algorithm
 )
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.asymmetric import (
     AsymmetricSignatureContext, AsymmetricVerificationContext, dsa
 )
 
 
-def _truncate_digest_for_dsa(dsa_cdata, digest, backend):
```

Comment:
The `_truncate_digest_for_ecdsa` doesn't have an equivalent comment, but could we check to see if that is no longer required? I just tested on 1.1.0 and it definitely does it, but hopefully ECDSA and DSA match up here.